examples/ex3p.cpp

```diff
@@ -70,6 +70,7 @@ int main(int argc, char *argv[])
    int order = 1;
    bool static_cond = false;
    bool pa = false;
+   bool nc = false;
```

Comment:
To ensure the new capability is tested, sample runs with `-nc` could be added in `ex3p` and `ex13p`.